src/uct/sm/scopy/cma/cma_md.c

```diff
@@ -27,6 +27,27 @@
 #endif
 
 
+typedef struct uct_cma_md {
+    struct uct_md       super;
+    uint64_t            extra_caps;
+} uct_cma_md_t;
+
+typedef struct uct_cma_md_config {
+    uct_md_config_t     super;
+    int                 mem_invalidate;
+} uct_cma_md_config_t;
+
+static ucs_config_field_t uct_cma_md_config_table[] = {
+    {"", "", NULL,
+     ucs_offsetof(uct_cma_md_config_t, super),
+     UCS_CONFIG_TYPE_TABLE(uct_md_config_table)},
+
+    {"MEMORY_INVALIDATE", "n", "Expose memory invalidate support capability",
```

Comment:
should it be yes/no/auto?

Reply:
what would "auto" mean?

Reply:
it we add cam invalidate later, it will not disable it

Reply:
or maybe document that this new config param means we expose invalidate support w/o actually supporting it?

Reply:
I guess we will have to delete this var when we add real support. 
So, add doc, but leave yes/no?

Reply:
ok